block_test.go

```diff
@@ -63,5 +64,6 @@ func TestBlockWithOddTransactions(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	fmt.Printf("%x\n", blk.Tx().Bytes())
+	_ = blk
+	// fmt.Printf("%x\n", blk.Tx().Bytes())
```

Comment:
Removed. It is _legacy_ code. I'm properly dealing with the tests for block in the next PR.